src/Components/Details.js

```diff
@@ -37,7 +37,7 @@ function Details() {
         {countries[index].capital}
       </p>
       <p className="card-text">
-        AREA IN SQ METRES :
+        AREA IN SQUARE METRES :
         {countries[index].area.toLocaleString()}
       </p>
       <p className="card-text">
```

Comment:
- [ ] Following the project requirements, you should follow the [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)) including:
  - Typography: font face, size, and weight
  - Layout: composition and space between elements.
<img width="704" alt="image" src="https://user-images.githubusercontent.com/77190203/182342824-bb5e4f8c-8d24-4445-ac93-f1567d1c2391.png">

- [ ] Your detail page design does not match the design guidelines. Please ensure that detail page design is the same as the one provided in the design guidelines. Checkout the comparison below 👇 :
- Your design:
<img width="206" alt="image" src="https://user-images.githubusercontent.com/77190203/182341904-de420367-b2df-4fca-8e76-ffb0d02b0c4a.png">

- The original design of the detail page:
<img width="128" alt="image" src="https://user-images.githubusercontent.com/77190203/182342219-1c7d708b-7c35-434c-a3ba-c06de9857d88.png">